cvpy/biomedimage/BiomedImage.py

```diff
@@ -0,0 +1,109 @@
+        self._cas_session = cas_session
+
+
+    def quantify_sphericity(self, image_table: CASTable, use_spacing: bool, input_background: float,
+                            label_connectivity: str, sphericity: CASTable) ->None:
+        ''''
+        Quantify the sphericity for the given component from a CAS table. 
+        Parameters:
+        ----------
+        image_table: CAS table
+             CAS table image includes the image binaries 
+        use_spacing: Bool
+             Specifies whether use spacing for the sphericity
+        input_background: float
+             Specifies the background value in input images.
+        label_connectivity:
+             Specifies the level of connectivity for connected components: LabelConnectivity.FACE or LabelConnectivity.VERTEX
+        sphericity: CASTable
+             CAS Output table
+        
+        Examples
+        --------
+        >>> ## Import classes
+        >>> from swat import CAS, CASTable
+        >>> from cvpy.biomedimage.BiomedImage import BiomedImage
+        >>> ## Connect to CAS
+        >>> s = swat.CAS("example.com", 5570)
+        >>> ## Construct Biomed object
+        >>> biomed = BiomedImage(s)
+        >>> ## Construct tables that are paramters to the quantify_sphericity API
+        >>> image_table = s.CASTable(...)
+        >>> output_table = s.CASTable(...)
+        >>> ## Call the API
+        >>> BiomedImage.quantify_sphericity(image_table,....,output_table)
+
+        '''
+        conn = self._cas_session
+       
+        ## Load the actionsets
+        conn.loadactionset('image')
+        conn.loadactionset('biomedimage')
+        conn.loadactionset('fedsql')
+        
+        ## Quantify the volume and perimeter of the given component.
+        conn.biomedimage.quantifybiomedimages(images= dict(table= image_table),
+                                              copyvars=["_path_"],
+                                              region="COMPONENT",
+                                              quantities=[dict(quantityparameters=dict(quantitytype="perimeter")),
+                                              dict(quantityparameters=dict(quantitytype="content",useSpacing = use_spacing))
+                                              ],
+                                              labelparameters=dict(labelType="basic",connectivity=label_connectivity),
+                                              inputbackground=input_background,
+                                              casout=dict(name='quantify', replace=True),
+                                              )
+
+        ## Compute sphericity based on perimeter and volume of the lesion
+        conn.fedsql.execdirect(
+            "create table sphericity {option replace=true} as "
+            "select _path_,_perimeter_,_content_, (power(pi(), 1.0/3.0) * power(6*_content_, 2.0/3.0))/_perimeter_ as "
+            "sphericity from quantify"
+            )
```

Comment:
I am assuming "quantify" is a temporary intermediate table created by this function. If that is correct, it should be deleted here.